fabric-object-builders-v0/src/main/java/net/fabricmc/fabric/api/entity/FabricEntityTypeBuilder.java

```diff
@@ -83,19 +83,42 @@ protected FabricEntityTypeBuilder(EntityCategory category, EntityType.EntityFact
 	}
 
 	/**
-	 * @deprecated Use {@link FabricEntityTypeBuilder#size(EntitySize)}
+	 * @deprecated Use {@link FabricEntityTypeBuilder#dimensions(float, float)}
 	 */
 	@Deprecated
 	public FabricEntityTypeBuilder<T> size(float width, float height) {
-		this.size = EntitySize.resizeable(width, height);
-		return this;
+		return dimensions(width, height);
 	}
 
+	/**
+	 * @deprecated Use {@link FabricEntityTypeBuilder#dimensions(EntitySize)}
+	 */
+	@Deprecated
 	public FabricEntityTypeBuilder<T> size(EntitySize size) {
-		this.size = size;
+		return dimensions(size);
+	}
+
+	/**
+	 * Defines the default {@link EntitySize} for this entity type
```

Comment:
What's the EntitySize used for ? Is it for rendering ? For collisions ? What do I put here ? I know it was not here before, but this is as good a time as any to add more information.